stob/linux.c

```diff
@@ -78,7 +78,7 @@
 
    @{
  */
-
+ 
```

Comment:
Blank line is OK. But please remove the space.

Reply:
removed the blank as well as space